src/main/java/com/spring/configuration/AddressDBConfiguration.java

```diff
@@ -0,0 +1,59 @@
+package com.spring.configuration;
+
+import javax.sql.DataSource;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceProperties;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.orm.jpa.JpaTransactionManager;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import com.spring.address.db.repositories.AddressRepository;
+
+
+@Configuration
+@EnableTransactionManagement
+@EnableJpaRepositories(basePackageClasses = AddressRepository.class, entityManagerFactoryRef = "addressFactoryBean", transactionManagerRef = "addressTransactionManager")
+public class AddressDBConfiguration {
+	
+
+	@Bean
+	@ConfigurationProperties("spring.datasource2")
+	public DataSourceProperties addressDataSourceProperties() {
+		return new DataSourceProperties();
+	}
+	
+
```

Comment:
Too many empty lines. Class methods should be split by 1 empty line. Change here and everywhere in the project.